src/node_crypto.cc

```diff
@@ -3320,6 +3320,7 @@ void Hmac::HmacDigest(const FunctionCallbackInfo<Value>& args) {
 
   bool r = hmac->HmacDigest(&md_value, &md_len);
   if (!r) {
+    delete[] md_value;
```

Comment:
@usta https://github.com/nodejs/node/pull/2872#discussion_r39479166 wasn't about the check _around_ the delete, it was about the check _including_ the delete.

To be clear: this `delete[]` isn't needed here, with or without checks. It should be removed.